product_docs/docs/edbcloud/beta/overview/02_high_availibility.mdx

```diff
@@ -2,21 +2,11 @@
 title: "High Availability"
 ---
 
-EDB Cloud provides built-in high availability aimed to reduce downtime in the event of a zone or instance becoming unavailable.
-An EDB Cloud _cluster_ is a PostgreSQL cluster made up of a single _primary_ and an optional number of _replicas_.
-Streaming replication is used to ensure that the primary and replicas have the same data.
+EDB provides optional high availability aimed to reduce downtime in the event of a zone or instance becoming unavailable. High availability clusters are configured with one _primary_ and two _replicas_ with synchronous streaming replication. Clusters are configured across availability zones in regions with availability zones.
```

Comment:
```suggestion
EDB provides optional high availability to minimize downtime in case of failures. High availability clusters are configured with three database instances - one _primary_ and two _replicas_ - with streaming replication from the primary to the replicas. In cloud regions with availability zones, instances are provisioned across multiple availability zones to provide fault tolerance in the face of datacenter failure.

Incoming client connections are always routed to the current primary. In case of failure of the primary, a standby replica will automatically be promoted to primary and new connections will be routed to the new primary. When the old primary recovers, it will re-join the cluster as a secondary.

 By default, transactions must synchronously commit to one replica before acknowledgement - in PostgreSQL terms, `synchronous_commit` is set to `on` and `synchronous_standby_names` is set to `ANY 1 (replica-1, replica-2)`. Customers can modify this behavior on a per-transaction, per-session, per-user, or per-database basis with appropriate `SET` or `ALTER` commands.
```